scopes/ui-foundation/use-box/menu/menu.compositions.tsx

```diff
@@ -3,11 +3,13 @@ import { Menu } from './menu';
 
 export const MenuExample = () => {
   return (
-    <Menu
-      packageName="@teambit/design.ui.input.radio"
-      componentId="teambit.design/ui/input/radio"
-      registryName="@teambit"
-      componentName="radio"
-    />
+    <div style={{ width: 400 }}>
+      <Menu
+        packageName="@teambit/design.ui.input.radio"
+        componentId="teambit.design/ui/input/radio"
+        registryName="@teambit"
+        componentName="radio"
+      />
+    </div>
```

Comment:
try to apply the style directly on the menu

Reply:
done